source/manual/alerts/content-publisher-healthcheck-not-okay.html.md

```diff
@@ -0,0 +1,32 @@
+---
+owner_slack: "#govuk-pubworkflow-dev"
+title: content-publisher app healthcheck not ok
+section: Icinga alerts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-02-24
+review_in: 6 months
+---
+
+There are a few things that this alert can be, you'll need to click on the alert to find out more details on what's gone wrong. There are more details and common fixes listed below.
+
+## Government Data Check
+
+This means that Content Publisher is having trouble updating the data it holds on current and previous governments. Ordinarily it reloads this data from the Publishing API every fifteen minutes and seeing this error means it hasn't happened in at least 6 hours. After 24 hours the cache of government data will clear and the app will stop working as expected. The following suggestions should help to isolate the problem.
+
+- Check [Sentry][] for any recent errors that indicate reasons the job is failing
+- Ensure there aren't alerts indicating the Content Publisher sidekiq process isn't running
+- Run `PopulateBulkDataJob.perform_now` manually in the [Content Publisher console][console] to see if issues occur
```

Comment:
Could we link to the class, to give a bit more context?

Reply:
Done :)